src/containers/UserInfo.tsx

```diff
@@ -0,0 +1,117 @@
+import { Card, Button, Row } from "react-bootstrap";
+import { GeoAlt, Link45deg, Balloon, Calendar3 } from "react-bootstrap-icons";
+import styled from "styled-components";
+
+const StyledCardBody = styled(Card.Body)`
+  height: 0px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+`;
+
+interface UserInfoInterface {
+  headLogo: string;
+  userLogo: string;
+  username: string;
+  flagLogo: string;
+  userNickname: string;
+  aboutMeText: string;
+  userLocation: string;
+  userEmail: string;
+  userBirthday: string;
+  registrationDate: string;
+  inReadersCount: number;
+  readersCount: number;
+}
+
+const UserInfo: React.FC<UserInfoInterface> = ({
+  headLogo,
+  userLogo,
+  flagLogo,
+  username,
+  userNickname,
+  aboutMeText,
+  userLocation,
+  userEmail,
+  userBirthday,
+  registrationDate,
+  inReadersCount,
+  readersCount,
+}) => {
+  return (
+    <Card className="rounded-0 border-0">
+      <Card.Img variant="top" className="rounded-0" src={headLogo} />
+
+      <StyledCardBody className="p-0">
+        <img
+          className="rounded-circle ms-3"
```

Comment:
move width and height to styled component

Reply:
is it necessary to create styled const only for width and height?